requirements.txt

```diff
@@ -2,4 +2,8 @@ numpy
 scipy
 Cython
 tqdm
-powerlaw
\ No newline at end of file
+powerlaw
+sentence_transformers
```

Comment:
We should remove the model dependencies. We're not going to install them by default.